docs-source/content/faq/boot-identity-not-valid.md

```diff
@@ -17,3 +17,13 @@ the domain directory, which occurs during the "introspection" phase, and when th
 Check that the user name and password credentials stored in the Kubernetes Secret referenced by `weblogicCredentialsSecret` contain the expected values for an account with administrative privilege for the WebLogic domain.
 Then [stop all WebLogic Server instances](https://oracle.github.io/weblogic-kubernetes-operator/userguide/managing-domains/domain-lifecycle/startup/#starting-and-stopping-servers)
 in the domain before restarting so that the operator will repeat its introspection and generate the corrected `boot.properties` files.
+
+If the "Boot identity not valid" error is still not resolved, Managed Server may be failing to communicate with the Administration Server for other reasons. Check carefully for any errors or exceptions logged before the "Boot identity not valid" error in the Managed Server log file and look for indications of communication failure reasons.
+
+If you enable SSL for the Administration Server and use the demo identity certificates, Managed Server may fail to establish an SSL connection with the Administration Server due to a hostname verification exception such as the "SSLKeyException: Hostname verification failed" error. For non-production environments, you can turn off the hostname verification by setting the "-Dweblogic.security.SSL.ignoreHostnameVerification=true" property in the Java options for starting the WebLogic Server.
```

Comment:
Put the above sentence, and the following sentence into a bulleted list that corresponds with the 'For example:' above.

Reply:
`If you enable SSL for the Administration Server and use the demo identity certificates, Managed Server may fail to establish an SSL connection with the Administration Server due to a hostname verification exception such as the "SSLKeyException: Hostname verification failed" error. For non-production environments, you can turn off the hostname verification by setting the "-Dweblogic.security.SSL.ignoreHostnameVerification=true" property in the Java options for starting the WebLogic Server.` 

-->

`If you have enabled SSL for the Administration Server default channel or have configured an admin port on the Administration Server, and the SSL port is using the default demo identity certificates, then a Managed Server may fail to establish an SSL connection to the Administration Server due to a hostname verification exception (such as a "SSLKeyException: Hostname verification failed" error). For non-production environments, you can turn off the hostname verification by setting the "-Dweblogic.security.SSL.ignoreHostnameVerification=true" property in the Java options for starting the WebLogic Server.`